lib/features/booking/add_inspection_booking_screen.dart

```diff
@@ -96,17 +99,26 @@ class AddInspectionBookingBody extends StatelessWidget {
                             )),
                       ],
                     ),
-                    BookedItemCard(
-                      propertyName: 'Green living space in Melbourne',
-                      propertyType: HatSpaceStrings.current.apartment,
-                      propertyImage:
-                          'https://m.media-amazon.com/images/I/81YR16yC2QL._AC_UF350,350_QL80_.jpg',
-                      price: 4800,
-                      state: 'Victoria',
-                      currency: Currency.aud,
-                      rentingPeriod: 'pw',
-                      onPressed: () {
-                        // TODO: implement BL
+                    BlocBuilder<PropertyDetailCubit, PropertyDetailState>(
+                      builder: (context, state) {
+                        if (state is PropertyDetailLoaded) {
+                          return BookedItemCard(
+                            propertyName: state.name,
+                            propertyType: state.type,
+                            propertyImage: state.photos.first,
+                            price: state.price.rentPrice,
+                            state: state.state,
+                            currency: state.price.currency,
+                            rentingPeriod: ' pw',
+                            onPressed: () {
+                              // TODO: implement BL
+                            },
+                          );
+                        }
+                        // TODO:handle when load property fail
+                        return const SizedBox(
+                          height: 100,
```

Comment:
Do u need height here?

Reply:
@suesitran, we don't need it, I already removed it, please check the latest commit. Thanks